### DISCUSSION TITLE
Is there a way to insert variables into the UI?

### DISCUSSION THREAD

**Discussion by TheTechTitanz** [FlowiseAI/Flowise#654] [Q&A]
I've tried everything!

I just want to push a variable from botpress to Flowise for a specific URL, and Flowise to look at that URL to feedback some information about the website.

However I've tried heaps of stuff to get this variable to work. I've called the variable myURL, and I've tried doing {{myURL}} @myURL {myURL}, but each time i get that the URL is not valid.

I'm using railway to host, and i've even tried setting variables on railway too, with no avail. Help!!

![image](https://github.com/FlowiseAI/Flowise/assets/131253089/4fa2328d-5865-449e-8601-978be80e6122)

![image](https://github.com/FlowiseAI/Flowise/assets/131253089/6202f7db-2e78-42de-b8c0-e8f6e813dd35)

**Comment by HenryHengZJ** [comment]
This is currently not possible, but we can add this feature 👍

**Comment by TheTechTitanz** [comment]
Please add it! I beg you. I was trying to figure this out all morning and
now I'm lying face down on the floor

**Reply by HenryHengZJ** [reply]
Actually you can pass the variable of the url to Flowise API like this:
```
{
    "question": "How to install Flowise?",
    "overrideConfig": {
        "url": "https://docs.flowiseai.com/getting-started"
    }
}
```

This will override the existing url you set on the chatflow, and use the one provided in `overrideConfig` instead

**Comment by TheTechTitanz** [comment]
Let me try this!

So pass the url variable through api, which overrides the existing url in
the UI (so I can just put whatever here if it looks legitimate), and will
use the external variable??? Sounds like a good solution to me.

**Comment by sirsimonson** [comment]
Is this also a valid workaround for https://github.com/FlowiseAI/Flowise/discussions/1664 ? 

In this case I would need to create the context variable via the "Variables" UI View and then just override it to i.e. use it as a promptValue?

**Reply by sirsimonson** [reply]
@HenryHengZJ Please take a tiny look at this. It's that little thing thats blocking me form reusing a chaining approach dynamically.
